dockerfiles/init/modules/che-kubernetes-helm/templates/configmap.yaml

```diff
@@ -44,3 +44,10 @@ data:
 {{- else }}
   CHE_INFRA_KUBERNETES_INGRESS_ANNOTATIONS__JSON: '{"nginx.ingress.kubernetes.io/rewrite-target": "/","nginx.ingress.kubernetes.io/ssl-redirect": "false","nginx.ingress.kubernetes.io/proxy-connect-timeout": "3600","nginx.ingress.kubernetes.io/proxy-read-timeout": "3600"}'
 {{- end }}
+{{- if .Values.isHostBased }}
+  CHE_INFRA_KUBERNETES_SERVER_STRATEGY: "host"
```

Comment:
Consider renaming var to `CHE_INFRA_KUBERNETES_SERVER__STRATEGY`. We have vars naming policy:
- dots in properties separate components (in case of env vars single underscores)
- underscores in properties separate words in a single component term (in case of env vars double underscores)

E.g.
che.component1.sub_component_2_mb -> CHE_COMPONENT1_SUB__COMPONENT__2__MB
where MB is unit of measurement in case when property is a number

Reply:
done